DuckDuckGo/Common/Extensions/NSAlertExtension.swift

```diff
@@ -43,6 +43,14 @@ extension NSAlert {
         return alert
     }
 
+    static func burnFireproofSiteAlert() -> NSAlert {
```

Comment:
This alert is migrated from SwiftUI (as seen in HomePage.Views.RecentlyVisitedView)